agent/kratos/kratos.yml

```diff
@@ -57,7 +57,7 @@ selfservice:
 log:
   level: info
   format: text
-  leak_sensitive_values: true
+  leak_sensitive_values: false
```

Comment:
is it related to this change?

Reply:
no but I noticed this is set to true and we probably dont want it that way